src/WebJobs.Extensions.ApiHub/Table/TableEntityValueBinder.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
```

Comment:
Can you enable StyleCop (see the example from the other projects in the repo)?  

---
In reply to: [64084638](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/98#discussion_r64084638) [](ancestors = 64084638)